test/hotspot/jtreg/runtime/cds/appcds/aotClassLinking/BulkLoaderTest.java

```diff
@@ -229,6 +232,29 @@ static void checkInitiatingLoader() throws Exception {
 
         throw new RuntimeException("Should not have succeeded");
     }
+
+    static void checkOldClasses() throws Exception {
+        // Resolve BadOldClassA from the constant pool without linking it.
+        // implNote: BadOldClassA will be excluded, so any resolved refereces
+        // to BadOldClassA should be removed from the constant pool.
+        Class c = BadOldClassA.class;
+        Object n = new Object();
+        if (c.isInstance(n)) {
```

Comment:
`// Note that type-testing BadOldClassA here neither loads nor initailizes it.`

Reply:
I added:
```
// Note that type-testing BadOldClassA here neither links nor initializes it.
                                                    ^^^^^
```